helm/keycloak/values/values-prod.yaml

```
@@ -45,9 +45,9 @@ probeInitialDelaySeconds: 180
 disableSsl: false
 
 resources:
-   requests:
-     cpu: 100m
-     memory: 4Gi
+  requests:
+     cpu: 1000m
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)